test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -325,29 +343,49 @@ std::string RandomCode::fillArguments(dev::eth::Instruction _opcode, RandomCodeO
 
 		switch (_opcode)
 		{
-		case dev::eth::Instruction::CREATE:
+		case eth::Instruction::EXTCODECOPY:
+			code += getPushCode(randUniIntGen() % 32);  //memstart2
```

Comment:
Would be much better to define special distributions (like maybe `MemAddressDist`, `MemLenDist`) and generators (`MemAddressGen`, `MemLemGen`) on top of the file and then not repeat `% 32` and `% 128` many times

(also this modulo operation results in non-uniform distribution)